status-payloads-spec.md

```diff
@@ -149,10 +141,91 @@ All incoming messages can be matched against a chat. Below you can find a table
 
 <!-- TODO: "group-user-message" is not complete. Does it require to explicitly join the group chat? Is there a way to invite someone? Also, if I start a new group chat (or join an existing one), I need to somehow calculate this chatID by myself. How to do it? -->
 
+### Contact Requests
+
+These messages are used to notify the receiving end that it has been added to the sender's contact. They are identified by the transit tags `c2`, `c3`, `c4` respectively, but they are all interchangeable, meaning a client SHOULD handle them in exactly the same way. 
+
+#### Payload
+
+Payload is a struct (a compound data type) with the following fields (order is important):
+
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | name | `string` | The self-assigned name of the user (DEPRECATED) |
```

Comment:
this is an ongoing thing... should we include things that are deprecated in v1?

Reply:
I think until we get around actually making sure that we can drop them yes, basically as I understand this field is not used anymore, but if you drop it a client might not accept the message as it fails validation, so unless we get around fixing for code freeze unfortunately we need to keep it as it is